content/docs/use-cases/versioning-data-and-model-files/index.md

```diff
@@ -30,9 +30,9 @@ on-premises storage (e.g. SSH, NAS) as well as any major cloud storage provider
 ## DVC is not Git!
 
 DVC metafiles such as `dvc.yaml` and `.dvc` files serve as placeholders to track
-data files and directories (among other purposes). They point to specific data
-contents in the <abbr>cache</abbr>, providing the ability to store multiple data
-versions out-of-the-box.
+the version of data files and directories (among other purposes). They point to
```

Comment:
OK I see you're going for "versioning data files" here — thanks for adding the summary of terms in the PR description.

In DVC we try to make an important distinction between tracking and versioning though (the latter mostly left to Git) so let's keep both? May be as simple as

```suggestion
data files and directories for versioning (among other purposes). They point to
```

Or something else (again, don't be afraid to add a sentence somewhere if that doesn't work).